### PR TITLE
190 frontend start existing workout functionality

### PR DESCRIPTION
**Pull request by L-Hen**
# What type of PR is this? (Check all that apply)

- [x] ✨**feature**: Introduces completely new code or new features.
- [ ] 🐛**fix**: Implements changes that fix a bug. Ideally, reference an issue if present.
- [ ] ♻️**refactor**: Includes any code-related change that is neither a fix nor a feature.
- [ ] ✅**build**: Encompasses all changes related to the build of the software, including changes to dependencies or the addition of new ones.
- [ ] ⚡️**test**: Pertains to all changes regarding tests, whether adding new tests or modifying existing ones.
- [ ] 🚰**ci**: Involves all changes related to the configuration of continuous integration, such as GitHub Actions or other CI systems.
- [ ] 📚**docs**: Includes all changes to documentation, such as README files, or any other documentation present in the repository.
- [ ] 🗑️**chore**: Captures all changes to the repository that do not fit into the above categories.

# Description

What did you change? How did you change it?
Added functionality to Start Existing Workout page + work on exercise list.

# Tests

How was this tested?

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [x] Manual tests
- [ ] Tests were NOT needed
- [ ] Other (explain below)

# [Optional] Screenshots
image

# Documentation

- [ ] Added to README.me
- [ ] Seperate document
- [ ] NO documentation needed

Link to external documentation:

# [Optional] Are there any post-deployment tasks we need to perform?
Add functionality to the next button and pass the desired workout to the next page.

### CODE REVIEW

File: apps/expo/src/components/workout/exerciseList.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <TouchableOpacity key={exercise.id} className='' 
+      style={{backgroundColor: selectExercise[exercise.id] ? "#48476D" : "#1E1E1E"}}
+      onPress={() => selectToggle(exercise.id)}>
+        <Text  className='text-slate-200 text-[20px] my-[12px]'>
+          {exercise.name} 
+        </Text>
+      </TouchableOpacity>)}
+  </ScrollView>
+
+  return (
```
Comment (elliotwesoff) [inline apps/expo/src/components/workout/exerciseList.tsx:33]: This is clean 👍

**Review by elliotwesoff — APPROVED** [review]

**Comment by thienguen** [comment]
Selected workout should highlight and should be sent to the following page when clicking "next"? This is implemented, right?

**Comment by taylorfinelli** [comment]
@thienguen not yet

**Comment by thienguen** [comment]
@taylorfinelli will it be done in this pr, or that would be henry next issue?

**Comment by mva919** [comment]
I will be taking it over

**Comment by thienguen** [comment]
lgtnm

**Comment by thienguen** [comment]
@L-Hen @taylorfinelli sync w main and I can hit the merge

**Review by mva919 — APPROVED** [review]
Looks good, would be nice if you added unit tests with your new files.

**Pull request merged**